src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -75,6 +79,14 @@ public org.folio.des.domain.dto.Job get(UUID id) {
     return entityToDto(jobDtoOptional.get());
   }
 
+  public Job getJobEntity(UUID id) {
+    Optional<Job> jobDtoOptional = repository.findById(id);
+    if (jobDtoOptional.isEmpty()) {
```

Comment:
Could you use orElseThrow method as this: jobDtoOptional.orElseThrow(() -> new NotFoundException(String.format("Job %s not found", id)))?